CDP.ts

```diff
@@ -37,8 +36,7 @@ export class CDP {
         proxy: undefined as string,
         ignoreCertError: false,
         verboseLog: false,
-        anonymousPaths: [] as RegExp[],
-        sendRequest: sendRequest
```

Comment:
revert all changes in this file..

Reply:
these changes we made together for sendRequest (applicable to e2e) - this was the 'create sendRequest abstract for SDK' step. If we remove sendRequest, I don't see how the integration with e2e will work..

Reply:
we did these together for the sendRequest changes when we started e2e, if we remove sendRequest then I don't see how the sdk integration into the e2e will work.